09_employment/compute_metrics_wage_ratio_2022_subgroup_new.do

```diff
@@ -0,0 +1,200 @@
+*because both 2022 and 2023 have full annual values, i use the annual here for both
+*because QECW is not available annually for 2023 yet, will use 2022 data so likely won't need this inflation
+
+replace wage = wage* 302.702/292.655 // 2023 CPI-U is first, then 2022 CPI-U	
+
+save "mit_living_wage-2023.dta", replace
+*/
+clear
+
+/***** import QCEW data *****/
+
+cd `raw'
+
+import delimited using "2022_data.csv", numericcols(14 15 16 17 18) varnames(1) clear
+
+/* keep industry level data and assign to divisions */
+keep if areatype == "County" // & industry != "10 Total, all industries"
+
+** recode industries into MM categories
+generate industry_type = .
+	replace industry_type = 1 if naics == 10 & own == 0 // all
+	replace industry_type = 2 if naics == 101 // goods producing
+	replace industry_type = 3 if naics == 1028 // public admin
+	replace industry_type = 4 if naics == 1021 // trade transit utilities
+	replace industry_type = 5 if naics == 1022 // info services
+	replace industry_type = 6 if naics == 1023 | naics == 1024 // financial, prof, biz services
+	replace industry_type = 7 if naics == 1025 // ed/health
+	replace industry_type = 8 if naics == 1026 | naics == 1027 // leisure/hospitality/other
+	
+** variable labels 
+label define industry_labels_mm 1 "All" 2 "Goods Producing" 3 "Public Administration" 4 "Trade, Transit, Utilities" 5 "Information Services" 6 "Professional Services" 7 "Education and Health" 8 "Leisure and Other"
+
+label values industry_type industry_labels_mm
+
+drop if industry_type == . // drop super-sections and "unclassified" naics codes
+
+** keep variables pre-merge
+
+keep st cnty annualaverageweeklywage annualaverageestablishmentcount industry_type
+
+rename st state
+
+rename cnty county
+
+destring state, replace // make numeric for merging with MIT data
+
+cd `wages'
+
+/* merge living wage and QCEW data */
+merge m:1 state county using mit_living_wage-2022.dta
+
+*tab county state if _merge == 1*/// two AK counties missing from MIT data
+
+/* drop statewide obs */
+drop if _merge == 1
+
+/* only keep 1 adult, 2 children row */
+*keep if adults == "1 Adult" & children == "2 Children" // moved this earlier in the code to avoid a m:m merge in line 78
+
+/* drop duplicates (two counties repeated) */
+duplicates drop
+
+/* convert living hourly wage to weekly */
+gen weekly_living_wage = wage * 40
+
+/* get ratio (main metric) */
+gen average_to_living_wage_ratio = annualaverageweeklywage/weekly_living_wage
+
+/* create data quality flag 
+per discussion with Greg, >= 30 is 1, <30 is 3 */
+gen wage_ratio_quality = 1 if annualaverageestablishmentcount >= 30 & annualaverageestablishmentcount != .
+replace wage_ratio_quality = 3 if annualaverageestablishmentcount < 30 & annualaverageestablishmentcount != .
+replace wage_ratio_quality = . if annualaverageestablishmentcount == .
```

Comment:
same comment as other do file to check whether the metric should also be set to missing

Reply:
See comment above.